src/views/self/view_weapons.cpp

```diff
@@ -111,6 +111,9 @@ namespace big
 
 		components::sub_title("CUSTOM_WEAPONS"_T);
 
+		ImGui::Checkbox("Custom Gun Restriction", &g.self.custom_weapon_stop);
```

Comment:
This option is misleading and doesn't do what it says it does, it fully disables all the custom weapons regardless of if they're out or not.